File: tools/templates/test/src/main/python/test6/modules.py
import smv

class M2(smv.SmvModule, smv.SmvOutput):
    def requiresDS(self):
        return [ smv.smvdataset.SmvExtDataSet("integration.test.test6.M1") ]

    def run(self, i):
        return i[ smv.smvdataset.SmvExtDataSet("integration.test.test6.M1") ]

File: tools/templates/test/src/main/scala/test5/modules.scala
package integration.test.test5

import org.tresamigos.smv._

object M2 extends SmvModule("") with SmvOutput {
  override def requiresDS = Seq(SmvExtModule("integration.test.test5.modules.M1"))

  override def run(i: runParams) = {
    i(SmvExtModule("integration.test.test5.modules.M1"))
  }
}
